app/views/movies/index.html.erb

```diff
@@ -8,6 +8,10 @@
   <% if policy(Hall).index? %>
     <%= link_to "Check halls", halls_path, class: "btn btn-outline-secondary"%>
   <% end %>
+
+  <% if policy(Discount).index? %>
```

Comment:
You have some conflicts in this PR. I am not sure, but probably it is due to this line. You added it already in previously merged commit. It was added as comment. Is it responsible for this conflict?

Reply:
Yes it is, I'm gonna resolve it in a second :)